pkg/proxy/winkernel/proxier.go

```diff
@@ -197,12 +197,12 @@ func (t DualStackCompatTester) DualStackCompatible(networkName string) bool {
 	hns, _ := newHostNetworkService()
 	networkName, err := getNetworkName(networkName)
 	if err != nil {
-		klog.ErrorS(err, "unable to determine dual-stack status %v. Falling back to single-stack")
+		klog.ErrorS(err, "unable to determine dual-stack status. Falling back to single-stack")
```

Comment:
Please change it to
"Unable to determine dual-stack status, falling back to single-stack"